command/pr_create.go

```diff
@@ -55,6 +55,11 @@ func prCreate(cmd *cobra.Command, _ []string) error {
 		return fmt.Errorf("must be on a branch named differently than %q", baseBranch)
 	}
 
+	_, err = api.PullRequestForBranch(client, headRepo, headBranch)
```

Comment:
Shouldn't we query baseRepo here, since the "base" repository is the one where all the PRs are listed in?
```suggestion
	_, err = api.PullRequestForBranch(client, baseRepo, headBranch)
```